ansible/ci-initial-setup.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Initial setup to enable administration with ansible.
+
+- hosts: all
```

Comment:
Which ever group is specific to that class of servers... e.g. `ci`, `ci-freebsd`,